web/themes/custom/bglobal/source/_patterns/01-molecules/hero/_hero.scss

```diff
@@ -0,0 +1,74 @@
+.hero__title.h2 {
+  font-size: 3rem;
+  line-height: 5rem;
+}
+
+.hero__subtitle {
+  line-height: 1.5;
+}
+
+.hero__container {
+  background-color: #EEEEEE;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 450px;
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+  align-content: center;
+  justify-content: center;
+}
+
+.hero__image__container {
+  grid-area: 1 / 1 / 1 / 4;
+}
+
+img.hero__image {
+  display: grid;
+  height: 100%;
+  width: 100%;
+  min-width: 350px; 
+  max-width: 100% !important;
+  object-fit: cover;
+  object-position: center 25%;
+}
+
+.hero__text__container {
+  display: grid;
+  grid-area: 1 / 1 / 2 / 4;
+  grid-template-rows: 1fr 1fr 1fr;
+  padding: 2em 0;
+}
+
+.hero__title {
+  color: white;
+  grid-area: 1 / 1 / 1 / 2;
+  margin: 0px;
+  margin-left: 16px; /* adjust left to account for box shadow on highlight */
+  max-width: 450px;
+}
+
+.hero__subtitle {
+  margin-left: 16px; /* adjust left to account for box shadow on highlight */
+  grid-area: 2 / 1 / 2 / 1;
+}
+
+.hero__title .highlight--blue, .hero__subtitle .highlight--blue {
+  background-color: rgba(0,58,112,1);
+  padding: 0.5rem 0;
+  -webkit-box-shadow: 1rem 0px 0px rgba(0,58,112,1), -1rem 0px 0px rgba(0,58,112,1);
+  box-shadow: 1rem 0px 0px rgba(0,58,112,1), -1rem 0px 0px rgba(0,58,112,1);
+  -webkit-box-decoration-break:clone;
+  -moz-box-decoration-break:clone; 
+  box-decoration-break: clone;
+}
+
+.highlight--blue {
+  color: white;
+  display: inline;
+  font-family: "Bitter";
```

Comment:
Just for reference, I think variable exists for this font family, along with fallbacks. (`$font-bitter` I think)

Reply:
I actually meant to change this to Open Sans, but since I'm converting the subtitle from a h2 to a p, I'm going to remove the font-family specification on this class.